.mailmap

```diff
@@ -264,6 +266,7 @@ Rohit Patil                  <rohit4change@yahoo.in>
 Rohit Patil                  <rohit4change@yahoo.in> <Quan@Aries.(none)>
 Roman Tolesnikov             <rtolesnikov@yahoo.com>
 Ryan Cooke                   <ryancooke86@gmail.com>
+Sam leidenuniv               <orionlee@yahoo.com>
```

Comment:
Are you sure leidenuniv is a real last name and not shortform of Leiden University?

Reply:
That should have been Sam Lee, not sure what happened there. Will fix up in a follow-up PR

Reply:
Haha... what autocorrect would do `lee -> leidenuniv`? 🤔